streams/src/main/java/org/apache/kafka/streams/StreamsConfig.java

```diff
@@ -382,7 +394,13 @@
                     40 * 1000,
                     atLeast(0),
                     ConfigDef.Importance.MEDIUM,
-                    REQUEST_TIMEOUT_MS_DOC);
+                    REQUEST_TIMEOUT_MS_DOC)
+            .define(PROCESSING_GUARANTEE_CONFIG,
+                    ConfigDef.Type.STRING,
+                    "at_least_once",
+                    in("at_least_once", "exactly_once"),
+                    Importance.MEDIUM,
```

Comment:
I think MEDIUM is fine -- my interpretation is: HIGH -> you must most likely change it; MEDIUM -> double check, but default is probably ok; LOW: default is most likely ok -- only consider if you have a good reason